src/m365/spo/commands/tenant/tenant-recyclebinitem-remove.spec.ts

```diff
@@ -56,6 +49,12 @@ describe(commands.TENANT_RECYCLEBINITEM_REMOVE, () => {
     sinon.stub(Cli, 'prompt').callsFake(async () => (
       { continue: false }
     ));
+    sinon.stub(spo, 'ensureFormDigest').resolves({
```

Comment:
I think we can add this in the `before` hook, can't we?

Reply:
Did you see this one @nicodecleyre?

Reply:
This one should go here 😄: 

It makes the tests fail with 'invalid request'
![image](https://github.com/pnp/cli-microsoft365/assets/35696168/0e316487-6ddb-46ca-a5da-b3547ab51169)

Reply:
Right, the restore did it, nice! Thank you!